doc/guides/source/UserGuide/Protection.rst

```diff
@@ -128,17 +128,14 @@ By default, all objects in Indico are INHERITING.
 Domain Control Policy
 ---------------------
 
-If an Indico object (category, event, session, contribution,
-material, file and link) is PUBLIC, we can restrict the access to
-users accessing Indico from some given IPs (these IPs could be like
-127.1 which means that every IP starting like this will be valid).
-
-If the Indico object is RESTRICTED, this checking will not be
-applied.
-
-If it is INHERITING, it will have the same access protection as its
-parent. Its access protection status will therefore change whenever
-the parent's access protection changes.
+Access to an Indico category or event can be restricted to a range of IP addresses.
+To do that, the *Protection mode* of the Indico object needs to be set to
+**Protected**. Like any other ACL entry, it can also be used when the Indico object
+is set to **Inheriting** and it inherits from a **Protected** object. Then, the desired
+IP network can be selected from the *Access control list (ACL)*.
+
+In the rest of the cases, if the *Protection mode* is set to **Public** or **Inheriting**,
```

Comment:
Just above it's mentioned that *Like any other ACL entry, it can also be used when the Indico object
 is set to* ***Inheriting*** *and it inherits from a* ***Protected object***. When I say *In the rest of the cases* this includes the case *when the Indico object is set to* ***Inheriting*** *and it inherits from a* ***NON Protected*** *object*